src/utils/ethereumUtils.ts

```diff
@@ -288,7 +289,8 @@ const getNetworkFromChainId = (chainId: number): Network => {
  */
 const getNetworkNameFromChainId = (chainId: number): string | undefined => {
```

Comment:
we don't use this util in anywhere else so im tweaking to use the display name(NetworkInfo) vs the underlying network name(chains.json)